<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QDialog" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(74,145,158);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="menu_employee">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(206,106,107);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>EMPLOYEES MANAGEMENT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_clients">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(190,211,195);
border-radius:10px;
    border-radius: 10px;
 </string>
   </property>
   <property name="text">
    <string>CLIENTS MANAGEMENT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_partenaire">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(235,172,162);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>PARTENAIRE MANAGEMENT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_transport">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(190,211,195);
border-radius:10px;
    border-radius: 10px;
 </string>
   </property>
   <property name="text">
    <string>TRANSPORT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="menu_missions">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(206,106,107);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>MISSIONS MANAGEMENT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>120</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MENU</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
